lib/wifi_iot.dart

```diff
@@ -19,6 +19,9 @@ const EventChannel _eventChannel =
     const EventChannel('plugins.wififlutter.io/wifi_scan');
 
 class WiFiForIoTPlugin {
+  /// Returns whether the WiFi AP is enabled or not
+  @Deprecated(
+      "This is will only work with < Android SDK 26. It could be make to work for >= Android SDK 29, request at https://github.com/alternadom/WiFiFlutter/issues/134.")
```

Comment:
minor typo in error message: `make` -> `made`

Reply:
Resolved in 7f5909a3b4dd935c522bd3a8888b7840f7ddeadf